chore: upgrade dependencies

marnixdessing opened the pull request:
Upgraded dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3954108612

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #84 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3963559499

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #85 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3972300189

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #86 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3977246274

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #87 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3982255778

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #88 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/3991780355

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #89 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4001638321

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #90 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4011117986

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #91 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4020344598

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #92 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4029215274

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #93 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4034251294

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #94 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4039368307

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #95 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4049447573

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #96 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4059613023

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #97 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4069988090

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #98 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4079873484

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #99 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4088966222

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #100 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4094099177

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #101 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4099174829

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #102 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4109195792

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #103 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4119484908

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #104 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4129651367

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #105 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4139623274

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #106 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4148730579

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #107 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4153821570

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #108 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4158960628

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #109 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4169130129

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #110 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4179327848

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #111 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4189392624

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #112 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4199429413

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #113 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4208644175

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #114 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4213690592

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #115 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4218889020

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #116 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4228068271

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #117 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4238084393

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #118 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4248197289

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #119 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4258077474

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #120 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4267347906

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #121 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4272504410

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #122 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4277680520

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #123 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4288168825

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #124 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4298692414

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #125 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4309284379

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #126 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4319483818

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #127 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4328344219

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #128 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4333536649

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #129 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4338743486

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #130 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4349298089

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #131 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

joostvanderborg reviewed (approved):
👍 Alleen dependency updates, draait in accp goed.

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-gegevens/actions/runs/4359839827

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

invalid-email-address committed:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #133 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies
